packages/server/postgres/getPgConfig.ts

```diff
@@ -6,7 +6,7 @@ const getPgConfig = () => ({
   database: process.env.POSTGRES_DB,
   host: process.env.POSTGRES_HOST,
   port: Number(process.env.POSTGRES_PORT),
-  max: PROD ? 40 : 5
+  max: PROD ? 30 : 5
```

Comment:
I might miss something, but the total max is 47, so having 30 per process can exhaust the pool with just 2 processes. Given we have 2 executors, 20 seems like a better choice here.